Specs/DateTime/English/DateTimeModel.json

```diff
@@ -20421,5 +20421,31 @@
         }
       }
     ]
+  },
+  {
+    "Input": "I'll be back within the next day and 5 hours",
+    "Comment": "The resolution of this case is wrong and needs to be fixed in DateTimePeriodParser",
+    "Context": {
+      "ReferenceDateTime": "2016-11-07T16:12:00"
+    },
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "within the next day and 5 hours",
+        "Start": 13,
+        "End": 43,
+        "TypeName": "datetimeV2.datetimerange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "(2016-11-07T16:12:00,2016-11-07T21:12:00,PT5H)",
```

Comment:
This timex is not incorrect as the period only shows 5h.

Reply:
But it does not take into account the "next day" part and so the "end" turns out to be on the same day of the "start" while it should be on the next day. 
On the contrary, an expression like "within the next 1 day and 5 hours" returns the correct timex (2016-11-07T16:12:00,2016-11-08T21:12:00,P1DT5H)

Reply:
Yeah... The timex generation/calculation is incorrect. But it's not really in the scope of this PR. Please open an issue to fix this and add it to the priority list. Thanks.